Automattic/woocommerce-payments
# Adjust WooPay custom message to be reused for T&C and privacy policy

Closes #6993

#### Changes proposed in this Pull Request

This PR updates WooPay's custom message field to allow adding T&C and Privacy Policy links using placeholders `[terms_of_service_link]` and `[privacy_policy_link]`. This also updates the default field value to:

> By placing this order, you agree to our `[terms_of_service_link]` and understand our `[privacy_policy_link]`.





#### Testing instructions





* Ensure you don't have any value set for this field by removing it from `woocommerce_woocommerce_payments_settings` in `wp_options`.
* Go to WooPay settings and see the updated field name, description, and default value.
* Update the description and try using the shortcodes.
* The shortcodes and the description will only link to their respective pages when this page is set. When the page is not set it will not be a link. So try different combinations of pages set/unset.

-------------------


**Post merge**
